src/renderer/components/menu.vue

```diff
@@ -1,22 +1,18 @@
 <template>
-    <div class="top-bar">
-        <div class="top-bar-left">
-            <ul class="menu">
-              <router-link tag="li" active-class="is-active" :to="{ name: 'home-page' }">
-                <a><i class="fas fa-home"></i>&nbsp;Home</a>
-              </router-link>
-              <router-link tag="li" active-class="is-active" :to="{ name: 'settings' }">
-                <a><i class="fas fa-cogs"></i>&nbsp;Settings</a>
-              </router-link>
-              <router-link tag="li" active-class="is-active" :to="{ name: 'about-page' }">
-                <a><i class="fas fa-info-circle"></i>&nbsp;About</a>
-              </router-link>
-            </ul>
-        </div>
-        <div class="top-bar-right">
-          <ul class="menu">
-            <li><a @click="event => openSite(event, this.helpLink)"><i class="fas fa-question-circle"></i>&nbsp;Help</a></li>
-          </ul>
+    <div class="ui huge secondary menu">
+        <router-link class="item" active-class="active" :to="{ name: 'home-page' }">
+            <i class="home icon"></i>&nbsp;Home
```

Comment:
Use `hover text`. I'll add it to the standard.

Reply:
Icons are good for small places like phone screen. This is running on the screen we have the space, use it.

Reply:
Not necessarily. Icons are for graphic indication, and texts here are mostly uneeded. This is a UX pattern, you can find examples for it in GitHub/lab, office, Facebook, and a lot of other websites, and even if there are only few options here, you want to keep them close together and the texts sperate them.